core/includes/update.inc

```diff
@@ -418,6 +418,23 @@ function update_variable_del($name) {
 }
 
 /**
+ * Merge any new default values into the active configuration.
+ *
+ * Used by the update.php script to apply new default config data
+ * from updated modules.
+ *
+ * @param array $modules
+ *   An array where the keys are the modules with pending updates and the
+ *   values are an array of configuration files that require updates.
+ */
+function update_config_defaults($modules, &$context) {
+  foreach($modules as $module_name => $module_data) {
+    config_update_default_config($module_name);
+    $context['message'] = 'Updated default settings for ' . $module_name . '.';
```

Comment:
This message should be translated through t() right?

It also looks like only one module will be able to set a message. Each time the `message` key is being overwritten by the next message.